### DISCUSSION TITLE
Osu!Lazer Mania for Mobile Touch input

### DISCUSSION THREAD

**Discussion by PringleKitten** [ppy/osu#27430] [Ideas]
As of now, it's changed so the skin is massive to fit the playing field and make it "easier", but now, no skin can easily be read unless it is a bar skin.
My idea is, instead, bring back the normal play field ui from the older version, and instead add tiles on the screen that can be touched to play. Reference from REAL FNF Mobile.

Current issue with new mobile UI: https://youtu.be/wrrEw2i8pXU?si=fBRinMrGjPVNyPvu

Example of the Idea: https://youtu.be/u41lUSn-wIg?si=EIABqBc-ukjOqqJD

**Comment by Vosnezki** [comment]
I know this is a duplicate of dozens of similar issue, **but** I see this can be a temporal fix for the current issue, why?

- **Current "widen" bar on osu!mania have a very big visibility issue.** Also to mention that a finger can cover basically half of the phone. This might be not the case for tablet but still.

- **Those who use keyboard or virtual button is suffering from current layout.** I see many people struggle to play because there's no toggle to revert to the old layout. This will force player to revert back to the old version, and of course, no pp given.

I saw some WIP progress of doing a 3D perspective bar, but unless this will be done soon, I think this should be implemented first for now. (or disable this widen bar if custom skin set)
